pkg/testutil/testutil.go

```diff
@@ -58,6 +58,7 @@ type Base struct {
 	Binary           string
 	Args             []string
 	Env              []string
+	Dir              string
```

Comment:
Need this to test relative paths.
We cannot `os.Chdir` for the same reasons we cannot manipulate the environment directly.